src/component/plasmic/shared/PlasmicHeader.tsx

```diff
@@ -243,7 +243,7 @@ function PlasmicHeader__RenderFunc(props: {
             )}
             href={
               hasVariant(variants, "views", "publicView")
-                ? ("http://venturemark.co/" as const)
```

Comment:
Any changes in `src/component/plasmic` will be overwritten next time you run `plasmic sync`. You'll need to make the change in Plasmic. This works to get it changed for now though.